server/index.js

```diff
@@ -1,11 +1,38 @@
 /* eslint linebreak-style: ["error", "windows"] */
 const morgan = require('morgan');
 const express = require('express');
+const db = require('./database');
 
 const app = express();
 const port = 3333;
 
-app.use(express.static('public'));
 app.use(morgan('dev'));
+app.use(express.static('public'));
+
+app.get('/api/:propertyId/basicdetails', (req, res) => {
+  // TODO: Query property database here and return property's basic details
+  // propertyId stored in req.params.propertyId
+  const propertyId = Number(req.params.propertyId);
+  db.getDetails(propertyId)
+    .then((details) => {
+      res.status(200).send(details);
+    })
+    .catch((error) => {
+      res.status(418).send(error); // i'm a teapot
+    });
+});
+
+app.get('/api/photos/:propertyId', (req, res) => {
```

Comment:
I know the code base is not very large here and how do you feel about breaking your server code into the MVC model such that this file would send the req.body to the controller and then send the data to the model and then the model would grab the data from the db and send it back down to the controller and back down to the server and so on.

Reply:
In general, the emphasis of this portion isn't the backend, so I chose not to focus too much on the server-side. Something simple that handles the api calls I need was enough for me. Good practice to do MVC though! If you want me to make those changes, I can do it.